x-pack/plugins/security_solution/public/timelines/components/netflow/index.test.tsx

```diff
@@ -136,300 +134,244 @@ const getNetflowInstance = () => (
       'United States'
     );
   });
 
   test('it renders source.geo.country_iso_code', () => {
-    const wrapper = mount(<TestProviders>{getNetflowInstance()}</TestProviders>);
+    render(<TestProviders>{getNetflowInstance()}</TestProviders>);
 
-    expect(wrapper.find('[data-test-subj="source.geo.country_iso_code"]').first().text()).toEqual(
+    expect(screen.getByTestId('draggable-content-source.geo.country_iso_code').textContent).toBe(
       'US'
     );
   });
 
   test('it renders source.geo.region_name', () => {
-    const wrapper = mount(<TestProviders>{getNetflowInstance()}</TestProviders>);
+    render(<TestProviders>{getNetflowInstance()}</TestProviders>);
 
-    expect(wrapper.find('[data-test-subj="source.geo.region_name"]').first().text()).toEqual(
+    expect(screen.getByTestId('draggable-content-source.geo.region_name').textContent).toBe(
       'Georgia'
     );
   });
 
   test('it renders source.geo.city_name', () => {
-    const wrapper = mount(<TestProviders>{getNetflowInstance()}</TestProviders>);
+    render(<TestProviders>{getNetflowInstance()}</TestProviders>);
 
-    expect(wrapper.find('[data-test-subj="source.geo.city_name"]').first().text()).toEqual(
+    expect(screen.getByTestId('draggable-content-source.geo.city_name').textContent).toBe(
       'Atlanta'
     );
   });
 
   test('it renders the source ip and port, separated with a colon', () => {
-    const wrapper = mount(<TestProviders>{getNetflowInstance()}</TestProviders>);
+    render(<TestProviders>{getNetflowInstance()}</TestProviders>);
 
-    expect(
-      removeExternalLinkText(wrapper.find('[data-test-subj="source-ip-and-port"]').first().text())
-    ).toContain('192.168.1.2:9987');
+    expect(screen.getByTestId('source-ip-badge').textContent).toContain('192.168.1.2:9987');
   });
 
   test('it renders source.packets', () => {
-    const wrapper = mount(<TestProviders>{getNetflowInstance()}</TestProviders>);
+    render(<TestProviders>{getNetflowInstance()}</TestProviders>);
 
-    expect(wrapper.find('[data-test-subj="source-packets"]').first().text()).toEqual('2 pkts');
+    expect(screen.getByText('2 pkts')).toBeInTheDocument();
   });
 
   test('it hyperlinks tls.client_certificate.fingerprint.sha1 site to compare the fingerprint against a known set of signatures', () => {
-    const wrapper = mount(<TestProviders>{getNetflowInstance()}</TestProviders>);
+    render(<TestProviders>{getNetflowInstance()}</TestProviders>);
 
-    expect(
-      wrapper
-        .find('[data-test-subj="client-certificate-fingerprint"]')
-        .find('[data-test-subj="certificate-fingerprint-link"]')
-        .first()
-        .props().href
-    ).toEqual(
+    expect(screen.getByText('tls.client_certificate.fingerprint.sha1-value')).toHaveAttribute(
+      'href',
       'https://sslbl.abuse.ch/ssl-certificates/sha1/tls.client_certificate.fingerprint.sha1-value'
     );
   });
 
-  test('renders tls.client_certificate.fingerprint.sha1 text', () => {
```

Comment:
This test was merged with the previous test since they were testing the same element.